opni-insights-service/app/endpoint_functions.py

```diff
@@ -580,32 +577,30 @@ async def get_namespace_breakdown(start_ts, end_ts):
             }
         },
         "aggs": {
-            "namespace_name": {
-                "terms": {"field": "kubernetes.namespace_name.keyword"},
-                "aggs": {
-                    "anomaly_level": {"terms": {"field": "anomaly_level.keyword"}}
-                },
+            "bucket": {
+                "composite": {
+                    "size": 50,
```

Comment:
any reason for size 50?

Reply:
I was trying to think of what would be a good number of results to come up at once. I suppose I can bump that number higher to 1000 or so.